### PR TITLE
Add id field to websocket response when chat message is created.

### PR DESCRIPTION
**Pull request by gibsonbailey**
# @ mention of reviewers
@ckcollab

# A brief description of the purpose of the changes contained in this PR.
Title says it all

# To be addressed in the future:
[Use serializer to send data to chat-message recipients](https://github.com/ckc-org/django-chit-chat/issues/11)

# Checklist
- [ ] Code review by me 
- [ ] Hand tested by me 
- [ ] I'm proud of my work
- [ ] Code review by reviewer
- [ ] Hand tested by reviewer
- [ ] Ready to merge

### CODE REVIEW

File: tests/integration/test_chat_async.py [hunk 1]
```diff
@@ -149,7 +150,13 @@ async def test_serializer_hook():
 
     resp = await communicator.receive_json_from()
     resp.pop('time')
-    assert resp == {'room': room.pk, 'text': 'hello', 'type': 'chat', 'user': user.pk}
+    assert resp == {
+        'room': room.pk,
+        'text': 'hello',
+        'type': 'chat',
+        'user': user.pk,
+        'id': (await get_messages_from_room(room))[-1].id
```
Comment (ckcollab) [inline tests/integration/test_chat_async.py:158]: what if this fails, or will it never fail?

Comment (gibsonbailey) [reply tests/integration/test_chat_async.py:158]: If this fails, we should know about it. It means there was no message object created, which is a problem.

Comment (gibsonbailey) [reply tests/integration/test_chat_async.py:158]: It should never fail if things are working as expected, unless I'm missing something of course.

Comment (ckcollab) [reply tests/integration/test_chat_async.py:158]: ugh my bad, didn't notice this was in a test ... ! ty

**Review by ckcollab — APPROVED** [review]

**Pull request merged**